src/module.ts

```diff
@@ -30,9 +31,7 @@ export default defineNuxtModule<ModuleOptions>({
   defaults: {
       server: {
         key: null,
-        options: {
-          apiVersion: '2022-11-15'
```

Comment:
👍

Reply:
Just for info I removed it as by default stripe will take the latest. But we can override it ofc